libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2002,19 +2002,24 @@ bool GCS_MAVLINK::sending_mavlink1() const
     return ((_channel_status.flags & MAVLINK_STATUS_FLAG_OUT_MAVLINK1) != 0);
 }
 
-#if AP_RC_CHANNEL_ENABLED
```

Comment:
This looks wrong.

We should avoid calling this method rather than putting stuff in the body.